src/components/loading/loading-controller.ts

```diff
@@ -35,6 +35,11 @@ import { LoadingOptions } from './loading-options';
  * `onDidDismiss` function can be called to perform an action after the loading
  * indicator is dismissed.
  *
+ * Attempting to dismiss a loading indicator before it has been completely
+ * presented will result in errors, so make sure to enclose all code paths
```

Comment:
Would "uncaught exception" be better? It takes down the app when running under `ionic serve`.